cypress.config.ts

```diff
@@ -22,66 +23,12 @@ export default defineConfig({
-        stubGetSentencingMigrationDetailsCompleted: nomisMigrationApi.stubGetSentencingMigrationDetailsCompleted,
-
-        stubListOfAppointmentsMigrationHistory: nomisMigrationApi.stubListOfAppointmentsMigrationHistory,
-        stubStartAppointmentsMigration: nomisMigrationApi.stubStartAppointmentsMigration,
-        stubGetAppointmentsFailures: nomisMigrationApi.stubGetAppointmentsFailures,
-        stubDeleteAppointmentsFailures: nomisMigrationApi.stubDeleteAppointmentsFailures,
-        stubGetAppointmentsMigrationEstimatedCount: nomisPrisonerApi.stubGetAppointmentsMigrationEstimatedCount,
-        stubGetAppointmentsMigrationDetailsStarted: nomisMigrationApi.stubGetAppointmentsMigrationDetailsStarted,
-        stubGetAppointmentsMigrationDetailsCompleted: nomisMigrationApi.stubGetAppointmentsMigrationDetailsCompleted,
-
-        stubListOfActivitiesMigrationHistory: nomisMigrationApi.stubListOfActivitiesMigrationHistory,
-        stubStartActivitiesMigration: nomisMigrationApi.stubStartActivitiesMigration,
-        stubGetActivitiesFailures: nomisMigrationApi.stubGetActivitiesFailures,
-        stubDeleteActivitiesFailures: nomisMigrationApi.stubDeleteActivitiesFailures,
-        stubGetActivitiesMigrationEstimatedCount: nomisMigrationApi.stubGetActivitiesMigrationEstimatedCount,
-        stubGetActivitiesMigrationDetailsStarted: nomisMigrationApi.stubGetActivitiesMigrationDetailsStarted,
-        stubGetActivitiesMigrationDetailsCompleted: nomisMigrationApi.stubGetActivitiesMigrationDetailsCompleted,
-        stubEndActivities: nomisMigrationApi.stubEndActivities,
-
-        stubListOfAllocationsMigrationHistory: nomisMigrationApi.stubListOfAllocationsMigrationHistory,
-        stubStartAllocationsMigration: nomisMigrationApi.stubStartAllocationsMigration,
-        stubGetAllocationsFailures: nomisMigrationApi.stubGetAllocationsFailures,
-        stubDeleteAllocationsFailures: nomisMigrationApi.stubDeleteAllocationsFailures,
-        stubGetAllocationsMigrationEstimatedCount: nomisMigrationApi.stubGetAllocationsMigrationEstimatedCount,
-        stubGetAllocationsMigrationDetailsStarted: nomisMigrationApi.stubGetAllocationsMigrationDetailsStarted,
-        stubGetAllocationsMigrationDetailsCompleted: nomisMigrationApi.stubGetAllocationsMigrationDetailsCompleted,
-
-        stubMigrationInProgress: nomisMigrationApi.stubMigrationInProgress,
-        stubMigrationInProgressCompleted: nomisMigrationApi.stubMigrationInProgressCompleted,
-
-        stubGetVisitMigrationRoomUsage: nomisMigrationApi.stubGetVisitMigrationRoomUsage,
-        stubGetVisitRoomMappings: mappingApi.stubGetVisitRoomMappings,
-        stubGetVisitRoomUsage: nomisPrisonerApi.stubGetVisitRoomUsage,
-        stubAddVisitsRoomMapping: mappingApi.stubAddVisitsRoomMapping,
+        ...auth,
+        ...manageUsersApi,
+        ...tokenVerification,
+        ...nomisMigrationApi,
+        ...nomisPrisonerApi,
```

Comment:
decided to switch these over to new style, but that did cause one or two issues e.g. previously renamed whilst mapping
```
        stubDeleteFailures: nomisMigrationApi.stubDeleteVisitsFailures,
```